config/GregTech/GregTech.cfg

```diff
@@ -299,6 +299,23 @@ undergroundfluid {
             S:Registry=helium-3
         }
 
+        molten_iron {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=30
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=800
```

Comment:
800 is quite high for a metal, id go with a lower value

Reply:
changed to 400